runtime/browser/runtime.cc

```diff
@@ -29,16 +29,45 @@
 #include "runtime/browser/native_app_window.h"
 #include "runtime/common/constants.h"
 #include "runtime/browser/preload_manager.h"
+#include "runtime/browser/notification_window.h"
 
 namespace runtime {
 
 namespace {
 
-static NativeWindow* CreateNativeWindow() {
+const char kCategoryAlwaysOnTop[] = "http://tizen.org/category/always_on_top";
+
+static NativeWindow::WindowCategory GetWindowCategory(
+    std::shared_ptr<const wgt::parse::CategoryInfoList> category_info_list) {
+
+  for (const auto& category : category_info_list->categories) {
+    if (strcmp(category.c_str(), kCategoryAlwaysOnTop) == 0)
+      return NativeWindow::WindowCategory::ALWAYS_ON_TOP;
+  }
+  return NativeWindow::WindowCategory::NORMAL;
+}
+
+static NativeWindow* CreateNativeWindow(NativeWindow::WindowCategory category) {
   SCOPE_PROFILE();
   // TODO(wy80.choi) : consider other type of native window.
-  auto cached = PreloadManager::GetInstance()->GetCachedNativeWindow();
-  NativeWindow* window = cached != nullptr ? cached : new NativeAppWindow();
+  NativeWindow* window;
+
+  switch (category) {
+    case NativeWindow::WindowCategory::NORMAL: {
+      auto cached = PreloadManager::GetInstance()->GetCachedNativeWindow();
+      window = cached != nullptr ? cached : new NativeAppWindow();
+    }
+    break;
+    case NativeWindow::WindowCategory::ALWAYS_ON_TOP: {
+      auto cached = PreloadManager::GetInstance()->GetCachedNativeWindow();
```

Comment:
GetCachedNativeWindow() was used release of unique_ptr
So, You should just use delete operation